Test-№2.1-from-14.12.2022/task1.c

```diff
@@ -0,0 +1,197 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+void reset(bool *existCheck)
+{
+    for (int i = 0; i < 3; ++i)
+    {
+        existCheck[i] = false;
+    }
+}
+
+void resetTemp(int *array)
```

Comment:
```suggestion
void resetNumberRepresentation(int *numberRepresentation)
```
Что за temp, покторый нужно переустановить?
Пока не прочтешь код, не поймешь